components/inputs/input-date-time.js

```diff
@@ -123,6 +124,8 @@ class InputDateTime extends LabelledMixin(SkeletonMixin(FormElementMixin(Localiz
 		this._namespace = 'components.input-date-time';
 		this._preventDefaultValidation = false;
 		this._timeOpened = false;
+
+		this._contextObserverController = new HtmlAttributeObserverController(this, 'data-timezone');
```

Comment:
One disadvantage here is that the fact that our Intl library is using the `data-timezone` attribute is leaking into core -- if we want to change how that works down the road, there'll be downstream changes required.

Poking around, Intl actually does have a way to add change listeners, since it's actually already listening for changes itself:
https://github.com/BrightspaceUI/intl/blob/main/lib/common.js#L126

So I think you should be able to import `getDocumentLocaleSettings` and call into that and accomplish the same thing.

Reply:
Ah good call - I'll give that a try.

Reply:
Currently the change listeners only get called if language or fallback language get changed so I'll have to update `intl` to also call the change listeners called when timezone changes. Thoughts on if the listeners should just be called whenever any attributes we care about in https://github.com/BrightspaceUI/intl/blob/main/lib/common.js#L153 change?

Reply:
I think that makes sense yeah. And hopefully given the way `ifrau` works they'll all "change" together with a single callback instead of multiple.

Reply:
I think that ends up being true: https://github.com/Brightspace/ifrau/blob/master/port/port.js#L43

ifrau schedules all of the sync tasks in one promise resolution, so I imagine the next event loop will pick up all of those changes at once.